wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/WorkspaceManager.java

```diff
@@ -83,23 +88,28 @@
   private final WorkspaceRuntimes runtimes;
   private final AccountManager accountManager;
   private final EventService eventService;
+  private final PreferenceManager preferenceManager;
   private final WorkspaceValidator validator;
   private final DevfileIntegrityValidator devfileIntegrityValidator;
+  private final Clock clock;
```

Comment:
what's the benefit of having a Clock here if you don't use it for testing?

Reply:
don't get that the problem with Clock, but ok will replace it